app/assets/stylesheets/users/profile_edit/action_buttons.scss

```diff
@@ -0,0 +1,23 @@
+.user-profile-edit__form {
+  .action-buttons {
```

Comment:
typically when you are doing BEM, you declare these at top level, meaning it shouldn't be underneath `.user-profile-edit__form`

Reply:
I want to be sure that this styling is just for the action buttons within the form. I don't want it some other place for now. How can I achieve that if I move it to top level?